### PR TITLE
Add Disney +

### PR DESCRIPTION
**Pull request by HugoLeBoennec**
I'm adding **Disney+** with:

1. Overview Page w/ List
2. Sync Page
3. Test
4. Some king of mutli lang Url support

First time doing a Fork and Pull Request, i will look into your revisions 😀

### CODE REVIEW

**Comment by HugoLeBoennec** [comment]
So test can't be run headless because you need to be log and geoloc in specific country (UK for testing)

**Comment by Celuni** [comment]
I'd love to have this approved.
Let me know if you need my disney account with active subscription for testing purposes and where to contact you if so.

**Comment by lolamtisch** [comment]
It is a bit more difficult as there are no anime on disney plus germany.

File: src/pages/list.json [hunk 1]
```diff
@@ -324,6 +324,11 @@
     "type": "anime",
     "name": "DesuOnline"
   },
+  {
```
Comment (lolamtisch) [inline src/pages/list.json:327]: Please dont commit changes to this file

**Comment by HugoLeBoennec** [comment]
> It is a bit more difficult as there are no anime on disney plus germany.

I use a VPN because for the moment my country too

**Review by lolamtisch — CHANGES_REQUESTED** [review]

**Comment by HugoLeBoennec** [comment]
How do I rollback the files not to be pushed @lolamtisch

**Comment by lolamtisch** [comment]
If you want to do it clean then cherry pick with force push.

**Comment by HugoLeBoennec** [comment]
I finally did It

**Comment by HugoLeBoennec** [comment]
But before merging, i saw that Disney juste add "season", so i need to integre it

**Pull request closed without merging**